keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -660,9 +716,9 @@ enhancement:
   cluster required to make on upgrade, in order to make use of the enhancement?
 -->
 
-- Windows: This implementation requires no backports for OS components. 
-- Kubernetes: [need feedback on]
-- Containerd: [need feedback]
+- Windows: This implementation requires no backports for OS components.
+- Kubernetes: No changes required outside of ensuring feature gates are set while feature is in development.
+- Containerd: Must run a version of containerd with privileged container support (targeting v1.5+).
```

Comment:
The KEP mentioned changes needed for OCI. Has that been done already?

Reply:
Not yet. We're waiting for the changes in github.com/microsoft/hcsshim to merge and looking for feedback here on what to name those fields.